webhook_server/libs/handlers/pull_request_handler.py

```
@@ -1488,7 +1556,9 @@ async def check_if_can_be_merged(self, pull_request: PullRequest) -> None:
             _labels = await self.labels_handler.pull_request_labels_names(pull_request=pull_request)
             self.logger.debug(f"{self.log_prefix} check if can be merged. PR labels are: {_labels}")
 
-            is_pr_mergable = await asyncio.to_thread(lambda: pull_request.mergeable)
+            is_pr_mergable = await github_api_call(
+                lambda: pull_request.mergeable, logger=self.logger, log_prefix=self.log_prefix
+            )
             self.logger.debug(f"{self.log_prefix} PR mergeable is {is_pr_mergable}")
```

Ruff (0.15.9)

[warning] Logging statement uses f-string

(G004)